mw-webapp/src/component/userCard/skill/Skill.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from "src/component/userCard/skill/Skill.module.scss";
+
+/**
+ * User card skill props
+ */
+interface SkillProps {
+
+  /**
+   * User's skill name
+   */
+  skillName: string;
```

Comment:
do you need onDelete callback here?

Reply:
Let's fix it in the task about compose one tag component instead of way tag and skill - issue #649